exporter/metric/internal/statsd/conn.go

```diff
@@ -220,31 +240,29 @@ func (e *Exporter) format(rec export.Record, buf *bytes.Buffer) error {
 			return err
 		}
 		for _, pt := range points {
-			e.format1(rec, pt, format, buf)
+			e.formatSingleStat(rec, pt, format, buf)
 		}
 
-		// TODO handle non-Points Distribution/MaxSumCount.
-
 	} else if sum, ok := agg.(aggregator.Sum); ok {
 		sum, err := sum.Sum()
 		if err != nil {
 			return err
 		}
-		e.format1(rec, sum, formatCounter, buf)
+		e.formatSingleStat(rec, sum, formatCounter, buf)
 
 	} else if lv, ok := agg.(aggregator.LastValue); ok {
 		lv, _, err := lv.LastValue()
 		if err != nil {
 			return err
 		}
-		e.format1(rec, lv, formatGauge, buf)
+		e.formatSingleStat(rec, lv, formatGauge, buf)
 	}
 	return nil
 }
 
-func (e *Exporter) format1(rec export.Record, val core.Number, fmtStr string, buf *bytes.Buffer) {
-	// For basic statsd syntax, see
-	// https://github.com/statsd/statsd/edit/master/docs/metric_types.md
+// formatSingleStat encodes a single item statsd of statsd data
```

Comment:
should it be 'single item of statsd data'?

Reply:
Fixed.